src/app/faq-page/faq-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FaqPageComponent } from './faq-page.component';
+
+describe('FaqPageComponent', () => {
+  let component: FaqPageComponent;
+  let fixture: ComponentFixture<FaqPageComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ FaqPageComponent ]
```

Comment:
Tests currently fail due to missing imports and declarations.

Reply:
Done